entries/b/bankaust.com.au

```diff
@@ -0,0 +1,20 @@
+{
+  "Bank Australia": {
+    "domain": "bankaust.com.au.json",
+    "url": "https://www.bankaust.com.au",
```

Comment:
```suggestion
```

The `url` element is not needed because going to `https://[domain]` resolves correctly.